numba/np/arrayobj.py

```diff
@@ -3274,6 +3285,7 @@ def constant_bytes(context, builder, ty, pyval):
 
 
 @lower_builtin(operator.is_, types.Array, types.Array)
+@lower_builtin(operator.is_, types.Record, types.Record)
```

Comment:
This is the function compiled by Numba.

Reply:
Thank you for helping me out! I will incorporate these changes as soon as possible🙂